feature-group-definitions/promises.yml

```diff
@@ -1,4 +1,4 @@
-name: Promises
+name: Promises (initial support)
```

Comment:
While we're here, should we change the ID to `promise`, like `array`/"Arrays (initial support)"?